### ISSUE TITLE
'module' object has no attribute 'open'

### ISSUE THREAD

**Issue by wochner** [rk700/PyMuPDF#110]
Using Python 2.7.13 64bit on Windows 7
installed PyMuPDF with pip

```
import fitz
doc = fitz.open("test.pdf")
```

returns AttributeError

```
AttributeErrorTraceback (most recent call last)
<ipython-input-6-b9c97ecc048a> in <module>()
----> 1 fitz.open(r"F:\FM_Konzeption\FM_Spreadprodukte\Fondsmanager\Wochner\Datenbank\Python\reporting\test.pdf")

AttributeError: 'module' object has no attribute 'open'
```

What am I doing wrong?

**Comment by JorjMcKie** [comment]
This looks like `__init__.py` was not executed. This is where lots of methods and their synonyms are being defined. Among them `fitz.open = fitz.Document`.
I will try and reproduce your situation.

**Comment by wochner** [comment]
Thanks for your help. Seems the issue is related to running fitz in Atom using a jupyter kernel. It worked fine running the above command from the command line. I cant tell why (I am using the same kernel as only one version of Python is installed), as I am new to Python, but never had this situation before. And yes, the files are in the folder fitz.

**Comment by JorjMcKie** [comment]
In the meantime I have been able to follow your installation steps. Took me a while, because my Py2.7-64 installation was broken and had to be completely erased, before a new install finally worked.

Anyway: I used `py -2.7 -m pip PyMuPDF-1.11.2-cp27-cp27m-win_amd64.whl --upgrade` which worked and let me open documents as it should ...

So, no enlightenment here.

In order to confirm, that `__init__.py` indeed is not being executed, issue `print fitz.__doc__` after the import. You then should see something like this:

```
>>> print(fitz.__doc__)
PyMuPDF 1.11.2: Python bindings for the MuPDF 1.11 library,
built on 2017-05-09 03:15:34
>>>
```

The doc string is built in the init.
